app/views/questions/show.html.erb

```diff
@@ -15,5 +15,7 @@
 <% end %>
 
 <% unless @question.is_closed %>
-  <a href="/questions/<%= @question.id %>/answer" class="btn btn-primary">Answer this Question</a>
+	<%= link_to new_question_answer_path(@question), class: "btn btn-primary" do %>Answer this Question<% end %>
```

Comment:
`<%= link_to "Answer this Question", new_question_answer_path(@question), class: "btn btn-primary"  %>` also works